docs/book/src/developer/testing.md

```diff
@@ -37,7 +37,20 @@ a fake infrastructure provider to allow test coverage for testing the interactio
 
 ## Running unit and integration tests
 
-Using the `test` target through `make` will run all of the unit and integration tests.
+Run `make test` to execute all unit and integration tests.
+
+<aside class="note">
+
+<h1>Tips</h1>
+
+When testing individual packages, you can speed up the test execution by running the tests with a local kind cluster.
+This avoids spinning up a testenv with each test execution. It also makes it easier to debug, because it's straightforward
```

Comment:
This is really great! Up and running in a second. Big improvement to running integration tests.

Reply:
One disclaimer. Not all our current tests are repeatable without errors (we have a few which e.g. create nodes with hard-coded names but never clean them up). But I'll open an issue to clean up those cases the next few days.